docs/sources/api/_index.md

```diff
@@ -609,6 +610,21 @@ running Loki on Kubernetes, `/ready` can be used as a readiness probe.
 
 In microservices mode, the `/ready` endpoint is exposed by all components.
 
+## Change log level at runtime
+
+```
+GET /log_level
+POST /log_level
+```
+
+`/log_level` returns the log level in case of `GET` and allows to change the log level at runtime in case of `POST`.
+
+Params:
+
+- `log_level`: A valid log level that can be passed as URL param (`?log_level=<level>`) or as form value in case of `POST`. Valid levels: [debug, info, warn, error]
```

Comment:
```suggestion
- `log_level`: A valid log level that can be passed as a URL param (`?log_level=<level>`) or as a form value in case of `POST`. Valid levels: [debug, info, warn, error]
```